lib/job-iteration/integrations/resque.rb

```diff
@@ -10,8 +10,14 @@ def initialize(*) # @private
         super
       end
     end
-    # The patch is required in order to call shutdown? on a Resque::Worker instance
-    Resque::Worker.prepend(ResqueIterationExtension)
+
+    # @private
+    module ::Resque
+      class Worker
```

Comment:
I had to nest these instead of using

```ruby
class ::Resque::Worker
```

otherwise Rubocop would complain...

I generated the docs locally though and confirmed that `@private` causes neither the `Resque` module, nor the `Resque::Worker` class to show up in the documentation.